dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -70,6 +69,22 @@ RUN apt-get update &&        \
 # Install a newer version of rsyslog from stretch-backports to support -iNONE
 RUN apt-get -y -t stretch-backports install rsyslog
 
+# Install redis-tools
+
+{% if CONFIGURED_ARCH == "armhf" %}
```

Comment:
Reviewer will ask to indent these lines, wud u pls do it

Reply:
done